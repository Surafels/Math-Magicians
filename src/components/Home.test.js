import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

test('should render home div', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
