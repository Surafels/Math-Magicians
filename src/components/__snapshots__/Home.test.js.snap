// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home div 1`] = `
<div
  className="home-div"
>
  <h1>
    Welcome to my page
  </h1>
  <p>
    My interactive and user-friendly calculator allows you to perform basic arithmetic operations with ease. From addition and subtraction to multiplication and division, My calculator provides accurate results, helping you with your calculations in no time.
  </p>
  <p>
    Be inspired by our curated collection of success quotes. Delve into the wisdom of accomplished individuals from various fields. These quotes provide motivation and guidance on your journey to success, empowering you to overcome challenges and reach your goals.
  </p>
  <p>
    Join  on Math Magician, where calculation and inspiration intersect. Discover the joy of numbers while embracing the wisdom of success. Start exploring now and unlock your mathematical prowess and inner potential.
  </p>
</div>
`;
